nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/model/InstitutionPoints.java

```diff
@@ -34,28 +31,6 @@ public List<CreatorAffiliationPoints> creatorAffiliationPoints() {
     return nonNull(creatorAffiliationPoints) ? creatorAffiliationPoints : List.of();
   }
 
-  @Override
-  public boolean equals(Object o) {
-    if (this == o) {
-      return true;
-    }
-    if (o == null || getClass() != o.getClass()) {
-      return false;
-    }
-    InstitutionPoints that = (InstitutionPoints) o;
-    return Objects.equals(institutionId, that.institutionId)
-        && Objects.equals(
-            DecimalUtils.adjustScaleAndRoundingMode(institutionPoints),
```

Comment:
I would think this precision based equals is here for a reason and doing default equals could have some unintended consequences. I guess @brinxmat have been involved with this part at some point and have input here.